browser/tor/sources.gni

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/tor/buildflags/buildflags.gni")
+
+brave_browser_tor_sources = []
+brave_browser_tor_deps = []
+
+if (enable_tor) {
```

Comment:
please move tests in `browser/tor` to `browser/tor/test` so we don't have `BUILD.gn` and `source.gni` for the same directory